tuning/sft_trainer.py

```diff
@@ -216,6 +218,12 @@ def train(
     file_logger_callback = FileLoggingCallback(logger)
     peft_saving_callback = PeftSavingCallback()
     callbacks = [aim_callback, peft_saving_callback, file_logger_callback]
+    try:
+        tc_callback = TrainerController(train_control_args, train_args)
+        callbacks.append(tc_callback)
+    except Exception as e:
+        logger.error(f'TrainerController callback was not enabled due to this exception: {e}')
+        exit(0)
```

Comment:
Why not just use logger.fatal?

Reply:
Removed the exit() call and changed the logger to warn. We dont need to exit the trainer in this case. If the callback doesnt initialize successfully, then training could still continue without issues.